app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -221,32 +221,32 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
           {
             key: -2,
             description: "Spanish",
-            url: `${handoutApiUrl}?lang=es`,
```

Comment:
changed the handout query param to handoutLanguage to disambiguate from the lang query param that is used by front-end for Google translate and WeGlot